src/view/com/composer/photos/ImageAltTextDialog.tsx

```diff
@@ -22,8 +22,6 @@ type Props = {
 export const ImageAltTextDialog = (props: Props): React.ReactNode => {
   return (
     <Dialog.Outer control={props.control}>
-      <Dialog.Handle />
```

Comment:
Removing all `Handle`s, because the native dialog includes a handle on both platforms.